yarn.lock

```diff
@@ -13053,7 +13053,7 @@ react-native-safe-area@^0.5.0:
   dependencies:
     "@types/react" "^16.8.8"
 
-react-native-sensitive-info@~5.4.0:
+react-native-sensitive-info@~5.4.2:
   version "5.4.2"
```

Comment:
It sems like this PR won't affect anything, because yarn.lock was loading 5.4.2 already anyway.